src/PKSim.Infrastructure/Services/DefaultIndividualRetriever.cs

```diff
@@ -46,8 +46,7 @@ public Individual DefaultIndividualFor(SpeciesPopulation speciesPopulation)
       {
          if (!_indvidualCacheProSpecies.Contains(speciesPopulation))
          {
-            var species = _speciesRepository.FindByName(speciesPopulation.Species);
-            var individualDTO = _individualDefaultValueRetriever.DefaultSettingForSpecies(species);
+            var individualDTO = _individualDefaultValueRetriever.DefaultSettingFor(speciesPopulation);
```

Comment:
This is now using the speciesPopulation instead of the species. Acutally makes sense